core/src/test/java/io/grpc/internal/ServerImplTest.java

```diff
@@ -354,7 +355,7 @@ public void methodNotFound() throws Exception {
     Metadata requestHeaders = new Metadata();
     StatsTraceContext statsTraceCtx =
         StatsTraceContext.newServerContext(
-            streamTracerFactories, "Waiter/nonexist", requestHeaders);
+            streamTracerFactories, null, "Waiter/nonexist", requestHeaders);
```

Comment:
please extract this to a member variable.  A lot of our functions have high parameter counts, so seeing `null` makes it hard to remember which param it was for.

Reply:
Done